### ISSUE TITLE
enable synchronization

### ISSUE THREAD

**Issue by developpeurSTS** [Odoo-mobile/framework#383]
Good evening dear all,

I have difficulties with setting the synchronization on the odoo mobile application.
By doing this,
Settings> Sync Settings> Odoo> user [Odoo8],
I get an empty list.
It is then impossible to eneable the synchronization of the models of the list.

Someone would have a solution to offer me?
Thank you

**Comment by ikerfah** [comment]
Hello, Did you add the services to your manifest ? ( you need to two tags , provider and service) , did you ?

**Comment by developpeurSTS** [comment]
Yes, everything is there.
When I run it on an emulator (android 4.4.2), I see that synchronization models appear normally.
But when I run it on a phone with the same characteristics as the emulator, I do not get anything.
I tried with 3 different phones with android versions (4.4.2, 7.0), same result.
Thank you

**Comment by ikerfah** [comment]
Can you post a screenshot for your manifest ? I want to see how did your wrote the provider and service

**Comment by developpeurSTS** [comment]
![capture1_emulateur](https://user-images.githubusercontent.com/44495233/47922775-c628b600-deaf-11e8-8c4c-f55a8f9c88bb.JPG)

that's the screenshot when the application is running on the emulator.
As you can see, I have the option to check / uncheck.

![screenshot_20181102-155046](https://user-images.githubusercontent.com/44495233/47922886-02f4ad00-deb0-11e8-9f3c-b243bd9fba94.jpg)

that's the screenshot when the application runs on my phone (Tecno Camon CM, android 7.0).
As you can see, I have nothing.


[Files.zip](https://github.com/Odoo-mobile/framework/files/2543022/Files.zip)

There you have the files that you asked me (provider, service, manifest).

**Comment by ikerfah** [comment]
This can be a issue from your device , you can find sync adapter for other apps ? if your can't, can you please go to  "Settings > Apps ... Show System" you should see "Calendar Storage" and must not be disabled.
and then enable it.

**Comment by developpeurSTS** [comment]
![screenshot_20181102-164847 1](https://user-images.githubusercontent.com/44495233/47925918-2ff88e00-deb7-11e8-9806-98b312d39805.jpg)

**Comment by developpeurSTS** [comment]
I just checked, it's activated, and all seems normal.

**Comment by ikerfah** [comment]
Right, Can you please add 'syncable' attribute to your provider and set it to true ?
android:syncable="true"

**Comment by developpeurSTS** [comment]
Sorry, I can not find the 'syncable' attribute anywhere. Can you guide me further?

**Comment by ikerfah** [comment]
inside your provider , 
`<provider
            android:name="com.odoo.addons.customers.providers.CustomersSyncProvider"
            android:authorities="${applicationId}.core.provider.content.sync.res_partner"
            android:label="@string/sync_label_customers"
            android:syncable="true"
            android:multiprocess="true" />`

**Comment by developpeurSTS** [comment]
Sorry nothing has changed.
